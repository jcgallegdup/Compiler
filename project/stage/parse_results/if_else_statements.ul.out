Insert key=Implement me! val=AST.Function@76ed5528
Insert key=Implement me! val=AST.Function@2c7b84de
Error:0(line):0(position): Could not find function with name 'main' and type 'void'.
