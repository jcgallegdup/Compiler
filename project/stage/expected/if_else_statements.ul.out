boolean f ()
{
    if (true)
    {
        println "ok";
    }
    else
    {
    }
}

void m ()
{
    if (1)
    {
    }
    else
    {
        if ("true")
        {
            ;
            return x;
        }
        else
        {
            if (1)
            {
                ;
                ;
            }
        }
    }
    if ("_01STRING")
    {
        print "WOOP";
    }
    else
    {
        print "WOOP";
    }
    if ('c')
    {
        println false;
        identifier;
    }
    else
    {
        if ("true")
        {
            ;
            return x;
        }
        else
        {
            if (1)
            {
                ;
                ;
            }
            if (id)
            {
                expr;
            }
            else
            {
                FALSE;
            }
        }
    }
    if (0.901)
    {
        if (true)
        {
            return ;
            return 'c';
        }
        else
        {
        }
        if (true)
        {
            return ;
            return 'c';
        }
    }
    else
    {
        println "d";
        print 1;
        if (1)
        {
        }
        else
        {
            if (true)
            {
                expr;
            }
        }
    }
    if (true)
    {
        ;
        ;
        id[1] = WOOP;
    }
    else
    {
        ;
        ;
        ;
        ;
    }
    if (ID)
    {
        identifier = "GOOD";
    }
    else
    {
    }
}
